composer.json

```diff
@@ -54,7 +54,8 @@
         "symfony/polyfill-intl-normalizer": "~1.0",
         "symfony/polyfill-mbstring": "~1.0",
         "symfony/polyfill-php83": "^1.27",
-        "symfony/polyfill-uuid": "^1.15"
+        "symfony/polyfill-uuid": "^1.15",
+        "ext-iconv": "*"
```

Comment:
I would remove the dep here and use the String component instead when you've used `iconv`.